matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/gossiping/WithHeldTests.kt

```diff
@@ -39,12 +41,14 @@ import org.matrix.android.sdk.internal.crypto.model.event.WithHeldCode
 
 @RunWith(AndroidJUnit4::class)
 @FixMethodOrder(MethodSorters.JVM)
+@LargeTest
```

Comment:
Where are effectively use this annotation `LargeTest` (and the other ones)? It's maybe not in this PR?

Reply:
Yes, we are not currently using those annotations. The plan is to start using this annotation to our tests when everything is annotated. I have annotated some of the tests but not all of them